Makefile

```diff
@@ -41,9 +41,14 @@ fixtures:
 	cd tests/generate_fixtures/; ./generate_target_anim_files.py ../fixtures/bad_alignments ../fixtures/bad_alignments/intermediates/ANIm
 	cd tests/generate_fixtures/; ./generate_target_dnadiff_files.py ../fixtures/viral_example ../fixtures/viral_example/intermediates/dnadiff
 	cd tests/generate_fixtures/; ./generate_target_dnadiff_files.py ../fixtures/bad_alignments ../fixtures/bad_alignments/intermediates/dnadiff
-	cd tests/generate_fixtures/; ./generate_target_dnadiff_matrices.py
+	cd tests/generate_fixtures/; ./generate_target_dnadiff_matrices.py ../fixtures/viral_example ../fixtures/viral_example/matrices
+	cd tests/generate_fixtures/; ./generate_target_dnadiff_matrices.py ../fixtures/bad_alignments ../fixtures/bad_alignments/matrices
 	cd tests/generate_fixtures/; ./generate_target_sourmash_files.py
-	cd tests/generate_fixtures/; ./generate_target_sourmash_matrices.py
+	cd tests/generate_fixtures/; ./generate_target_sourmash_matrices.py 
+	cd tests/fixtures/bad_alignments/matrices/; cp ./dnadiff_sim_errors.tsv ./ANIm_sim_errors.tsv
+	cd tests/fixtures/bad_alignments/matrices/; cp ./dnadiff_identity.tsv ./ANIm_identity.tsv
+	cd tests/fixtures/bad_alignments/matrices/; cp ./dnadiff_coverage.tsv ./ANIm_coverage.tsv    
+	cd tests/fixtures/bad_alignments/matrices/; cp ./dnadiff_aln_lengths.tsv ./ANIm_aln_lengths.tsv
```

Comment:
e.g.

```
ln -s ./dnadiff_aln_lengths.tsv ./ANIm_aln_lengths.tsv
```

Reply:
Quick two questions:
1. Why is `ln -s` a better option than `cp` in this case? I'm happy to change it, just asking out of curiosity.
2. I'm aware of re-writing history and how it can be useful in some cases, but I don't really understand the process. I don't want to make even more of a mess. Would you mind going over this with me on Monday? This PR can wait anyway.

Reply:
In this case the files are small, so the disk saving is marginal. However having this as a sym-link makes it very explicit that the contents of ``ANIm_aln_lengths.tsv`` are identical to  ``dnadiff_aln_lengths.tsv``.

I'd be happy to do a live git workthough on this with you on Monday, using ``git rebase origin/main -i`` with the ``-i`` for interactive.